label.rb

```diff
@@ -0,0 +1,12 @@
+class Label
```

Comment:
- [ ]  You're missing `attr_reader :item`, without this line you will get an error when trying to add a label to a book, kindly add this line of code.

```suggestion
class Label
attr_reader :items

```